settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
 
 INSTALLED_APPS = [
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
     "db",
 ]
+
+AUTH_USER_MODEL = "db.User"
```

Comment:
The AUTH_USER_MODEL setting should use the get_user_model() function instead of directly referencing the model string. This is a best practice to ensure compatibility and flexibility in your Django project.